client/src/components/PeopleView/PeopleView.js

```diff
@@ -24,6 +24,10 @@ export default class PeopleView extends Component {
     handleClick = (e, p) => this.setState({isShowingModal: true, selectedEmployee: e})
     handleClose = () => this.setState({isShowingModal: false})
 
+    componentWillMount() {
+        setChangeWeekTimeout(moment().endOf('week'), this.props.entryActions.resetWeek);
```

Comment:
It would be prettier to give startOf next week as a param here rather than giving endOf this week and adding 1 to it in the setChangeWeekTimeout function.